packages/grafana-e2e-selectors/src/selectors/components.ts

```diff
@@ -15,7 +15,11 @@ export const Components = {
     fromField: 'TimePicker from field',
     toField: 'TimePicker to field',
     applyTimeRange: 'data-testid TimePicker submit button',
-    calendar: 'TimePicker calendar',
+    calendar: {
+      label: 'TimePicker calendar',
+      openButton: 'Open TimePicker calendar',
+      closeButton: 'Close TimePicker calendar',
```

Comment:
2 questions here:
- is `TimePicker` something that makes sense to an end user, or should this just be `Calendar`/`Open calendar`/`Close calendar`?
- is this file supposed to hold user facing strings, or is it just for e2e selectors?

Reply:
- I agree. Open calendar makes more sense
- Both I think
maybe @hugohaggmark might have some thoughts

Reply:
> is TimePicker something that makes sense to an end user, or should this just be Calendar/Open calendar/Close calendar?

Thinking about it now...I feel it makes sense for us to provide as much context as we can to the user. In this case, saying something like "open timePicker calendar" might make more sense.

Reply:
yeah good point, i'm just not sure `TimePicker` is something we should surface to the user since it's the internal name of the component. the `aria-label` for the whole `TimePicker` is:
> Time range picker with current time range Last 6 hours selected

so maybe either:
- `Time range picker calendar`/`Open time range picker calendar`/`Close time range picker calendar`
- `Time range calendar`/`Open time range calendar`/`Close time range calendar`

wdyt? 🤔